test/ui-test/testSuites/suite_communities/tst_communityFlows/test.feature

```diff
@@ -11,133 +11,104 @@
 
 Feature: Status Desktop community
 
-    As a user I want to create a community and chat
+    As an admin user I want to create a community and do some action in the community
 
     The following scenarios cover basic flows of a community
 
-    Background:
-        Given A first time user lands on the status desktop and generates new key
-        When user signs up with username "tester123" and password "TesTEr16843/!@00"
-        Then the user lands on the signed in app
-        When the user opens the community portal section
-        Then the user lands on the community portal section
-
-	@merge
-    Scenario Outline: User creates a community
-        When the user creates a community named <community_name>, with description <community_description>, intro <community_intro> and outro <community_outro>
-        Then the user lands on the community named <community_name>
-
-        Examples:
-            | community_name    | community_description   | community_intro  | community_outro  |
-            | testCommunity1    | Community tested 1      | My intro for the community | My community outro  |
-
+ 	The feature start sequence is the following (setup on its own `bdd_hooks`):
+    ** given A first time user lands on the status desktop and generates new key
+    ** when user signs up with username "tester123" and password "TesTEr16843/!@00"
+    ** and the user lands on the signed in app
 
-	@merge @mayfail
-    Scenario Outline: Admin creates a community channel
-        When the user creates a community named myCommunity, with description My community description, intro Community Intro and outro Community Outro
-        Then the user lands on the community named myCommunity
-        When the admin creates a community channel named <community_channel_name>, with description <community_channel_description> with the method <method>
-        Then the user lands on the channel named <community_channel_name>
+    Background:
+        Given the user opens the community portal section
+        And the user lands on the community portal section
+        And the user creates a community named "myCommunity", with description "My community description", intro "Community Intro" and outro "Community Outro"
+        And the user lands on the community named "myCommunity"
 
+	@mayfail
+	# TODO: Scenario broken since in the action of creating a channel `mainWindow_CommunityColumnView_CommunityColumnView` object is not ready.
+    Scenario Outline: The admin creates a community channel
+        When the admin creates a community channel named "<community_channel_name>", with description "<community_channel_description>" with the method "<method>"
+        Then the user lands on the channel named "<community_channel_name>"
```

Comment:
Consider revising use of "lands on" across features to remove ambiguity. Example:

```suggestion
        Then the channel named "<community_channel_name>" is open
```